CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* When creating a class that extends Realm.Object and pushing the instantiated object to a list, a segmentation fault would occur.  This has been fixed by a null check and throwing an exception.
+* Creating an object from an instance of Realm.Object that was manually constructed (detached from Realm) would fail the second time.  Now we throw an meaningful exception the first time.
```

Comment:
Should we mention that these are temporary solutions?

Reply:
That's a valid point

Reply:
I added a parent bullet point to explain our current situation and that these are temporary fixes to avoid crashes.